presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemorySmoke.java

```diff
@@ -119,6 +119,9 @@ public void testCreateTableInNonDefaultSchema()
 
         assertQueryResult(format("SELECT count(*) FROM %s.schema1.nation", CATALOG), 13L);
         assertQueryResult(format("SELECT count(*) FROM %s.schema2.nation", CATALOG), 12L);
+
+        assertUpdate(format("DROP SCHEMA %s.schema1", CATALOG));
+        assertUpdate(format("DROP SCHEMA %s.schema2", CATALOG));
```

Comment:
\+ `assertUpdate(format("DROP SCHEMA IF EXISTS %s.schema2", CATALOG));`

Reply:
although having a dedicated test for create/drop schema would be even better, covering dropping schemas with and without tables